src/Common/ThreadStatus.cpp

```diff
@@ -24,9 +24,7 @@ namespace ErrorCodes
     extern const int LOGICAL_ERROR;
 }
 
-
-thread_local ThreadStatus * current_thread = nullptr;
-thread_local ThreadStatus * main_thread = nullptr;
```

Comment:
I've removed `main_thread` since it was unused AFAICS (and nothing complained)

Reply:
Indeed, `MainThreadStatus::main_thread` is used instead.